src/display.jl

```diff
@@ -244,17 +244,16 @@ function Base.show(io::IO, m::MIME"text/markdown", fig::FigureLike)
     throw(MethodError(show, io, m, fig))
 end
 
-function Base.show(io::IO, m::MIME, figlike::FigureLike)
+function Base.show(io::IO, m::MIME, figlike::FigureLike; backend = current_backend(), update_state=true)
```

Comment:
I think just `update` would be better since that's what we use in `display()`

Reply:
oh, thanks!  i didn't notice that.  changed in a new commit.